tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionDetailRepositoryTests.cs

```diff
@@ -521,11 +484,6 @@ await CatalogueItemEntityBuilder
 
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
-                .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId(Solution1Id)
                 .WithImplementationTimescales(expectedResult)
                 .Build()
                 .InsertAndSetCurrentForSolutionAsync();
```

Comment:
As above, we should call `InsertAsync` instead of `InsertAndSetCurrentForSolutionAsync`.

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`